.pipelines/.vsts-vhd-builder-windows.yaml

```diff
@@ -4,7 +4,7 @@
 # - BUILD_POOL - Azure DevOps build pool to use
 # - CLIENT_ID - Service principal ID
 # - CLIENT_SECRET - Service principal secret
-# - COPY_VHD - Set to 'True' if VHD should be copied to classic storage account in preparation for publishing
+# - DRY_RUN - Set to 'False' if VHD should be copied to classic storage account in preparation for publishing
```

Comment:
Set to True for testing

Reply:
`DRY_RUN` is set to be `False` in pipeline variables. Here is for comment only.